src/core/services/compiler/compiler.js

```diff
@@ -112,12 +112,19 @@ function mdCompilerService($q, $templateRequest, $injector, $compile, $controlle
         link: function link(scope) {
           locals.$scope = scope;
 
-          //Instantiate controller if it exists, because we have scope
+          // Instantiate controller if it exists, because we have scope
           if (controller) {
-            var invokeCtrl = $controller(controller, locals, true, controllerAs);
+
+            var ctrlLocals = angular.extend(locals, {
+              $element: element
```

Comment:
That's unfortunately not possible, because the `Attributes` constructor is internal-only.

> https://github.com/angular/angular.js/blob/master/src/ng/compile.js#L1508

Also doesn't make any sense to have the `$attrs` here, because the template is always static, and the user can't specify attributes dynamically.